stl/inc/future

```diff
@@ -1075,7 +1075,7 @@ public:
 
     shared_future(future<void>&& _Other) noexcept : _Mybase(_STD forward<_Mybase>(_Other)) {}
 
-    shared_future& operator=(shared_future&& _Right) {
+    shared_future& operator=(shared_future&& _Right) noexcept {

```

Comment:
`noexcept` in this line is needed by the standard, see [[futures.shared.future]](https://eel.is/c++draft/futures.shared.future). But it seems better to make this move-assignment operator (and the corresponding overload in the primary template) defaulted.

Reply:
I'm working on this in my own PR.